src/main/java/frc/robot/commands/shooter/ShootSpeaker.java

```diff
@@ -31,6 +32,19 @@ public boolean isFinished() {
         return (timer.hasElapsed(Constants.SHOOTER_TIME_AFTER_TRIGGER)) && (activated);
     }
 
+    @Override
+    public void execute() {
+        if (((RobotContainer.isRedAlliance() == true) && (((((drivetrain.getGyroAngle().getDegrees()) % 360) + 360) % 360) > 180)) || ((RobotContainer.isRedAlliance() == false) && (((((drivetrain.getGyroAngle().getDegrees()) % 360) + 360) % 360) < 180))) {
```

Comment:
Why do you need to add 360 and then mod 360 again? Is it because it could be negative degrees and it's being ensured it's positive?

Reply:
OK I just threw this into Excel and as far as I can tell, +360 then %360 makes no difference to the number.